optaplanner-examples/src/test/java/org/optaplanner/examples/cheaptime/app/CheapTimeSolveAllTurtleTest.java

```diff
@@ -32,4 +34,9 @@ protected CommonApp<CheapTimeSolution> createCommonApp() {
     protected Class<CheapTimeEasyScoreCalculator> overwritingEasyScoreCalculatorClass() {
         return CheapTimeEasyScoreCalculator.class;
     }
+
+    @Override
+    protected boolean supportsBavet() {
+        return false;
+    }
```

Comment:
One thing that just crossed my mind: `batchscheduling` does support neither CS-D nor CS-B. (It is a community-contributed example that never had a CS.) Not sure if the turtle tests execute that, but if they do, they should not.